src/views/Pages/Page404/Page404.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { Button, Col, Container, Input, InputGroup, InputGroupAddon, InputGroupText, Row } from 'reactstrap';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Button' is defined but never used.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)